pkg/asset/tls/aggregator.go

```diff
@@ -161,7 +161,7 @@ func (a *AggregatorClientCertKey) Generate(dependencies asset.Parents) error {
 		Validity:     ValidityOneDay,
 	}
 
-	return a.SignedCertKey.Generate(cfg, ca, "apiserver-proxy", DoNotAppendParent)
+	return a.SignedCertKey.Generate(cfg, ca, "aggregator-client", DoNotAppendParent)
```

Comment:
FYI, the same old name was already generated here https://github.com/openshift/installer/pull/1526/files#diff-8d9eb2f3ce41c8ad81321b7dc4d554d8R71 and that's why we had duplicate assets in #1468